patches/nodejs-mobile-react-native+0.6.3.patch

```diff
@@ -25,6 +25,15 @@ index e54f25c..178b15a 100644
          main.assets.srcDirs += '../install/resources/nodejs-modules'
      }
  
+@@ -239,7 +242,7 @@ if ("1".equals(shouldRebuildNativeModules)) {
+     GenerateNodeProjectAssetsLists.dependsOn "ApplyPatchScriptToModules"
+ 
+     def nativeModulesABIs = android.defaultConfig.ndk.abiFilters;
+-    if (nativeModulesABIs == null) {
++    if (!nativeModulesABIs) {
```

Comment:
somewhat lazy approach as null and empty collections in groovy are both falsy http://groovy-lang.org/semantics.html#the-groovy-truth

Reply:
Huh, well that makes it quite easy. I like the idea of "groovy truth"